nexus_transporter/CMakeLists.txt

```diff
@@ -94,27 +104,46 @@ target_compile_features(nexus_transporter_component INTERFACE cxx_std_17)
 rclcpp_components_register_node(nexus_transporter_component
   PLUGIN "nexus_transporter::TransporterNode"
   EXECUTABLE nexus_transporter_node
-  EXECUTOR SingleThreadedExecutor)
-
+  EXECUTOR MultiThreadedExecutor)
```

Comment:
Is this still needed (and does it actually do what it's supposed to? I remember you mentioned that it wasn't as simple as setting the executor in the CMake file)